2021-02-06 06:33:32,942 INFO    MainThread:8733 [wandb_setup.py:_flush():70] setting env: {}
2021-02-06 06:33:32,942 INFO    MainThread:8733 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver
2021-02-06 06:33:32,942 INFO    MainThread:8733 [wandb_setup.py:_flush():70] setting login settings: {}
2021-02-06 06:33:32,942 INFO    MainThread:8733 [wandb_setup.py:_flush():70] setting login settings: {}
2021-02-06 06:33:32,942 INFO    MainThread:8733 [wandb_init.py:_log_setup():318] Logging user logs to /home/iiitd/Desktop/Ahmad/doa1/wandb/run-20210206_063332-rbx5oayf/logs/debug.log
2021-02-06 06:33:32,942 INFO    MainThread:8733 [wandb_init.py:_log_setup():319] Logging internal logs to /home/iiitd/Desktop/Ahmad/doa1/wandb/run-20210206_063332-rbx5oayf/logs/debug-internal.log
2021-02-06 06:33:35,512 INFO    MainThread:8733 [wandb_run.py:_console_start():1308] atexit reg
2021-02-06 06:33:35,513 INFO    MainThread:8733 [wandb_run.py:_redirect():1175] redirect: SettingsConsole.REDIRECT
2021-02-06 06:33:35,513 INFO    MainThread:8733 [wandb_run.py:_redirect():1178] Redirecting console.
2021-02-06 06:33:35,513 INFO    MainThread:8733 [redirect.py:install():205] install start
2021-02-06 06:33:35,515 INFO    MainThread:8733 [redirect.py:install():220] install stop
2021-02-06 06:33:35,515 INFO    MainThread:8733 [redirect.py:install():205] install start
2021-02-06 06:33:35,515 INFO    MainThread:8733 [redirect.py:install():220] install stop
2021-02-06 06:33:35,515 INFO    MainThread:8733 [wandb_run.py:_redirect():1222] Redirects installed.
2021-02-06 06:36:00,234 INFO    MainThread:8733 [wandb_run.py:_atexit_cleanup():1278] got exitcode: 1
2021-02-06 06:36:00,234 INFO    MainThread:8733 [wandb_run.py:_restore():1250] restore
2021-02-06 06:36:00,234 INFO    MainThread:8733 [redirect.py:uninstall():224] uninstall start
2021-02-06 06:36:00,235 INFO    MainThread:8733 [redirect.py:_stop():279] _stop: stdout
2021-02-06 06:36:00,235 INFO    MainThread:8733 [redirect.py:_stop():285] _stop closed: stdout
2021-02-06 06:36:00,236 INFO    stdout    :8733 [redirect.py:_pipe_relay():121] relay done saw last write: stdout
2021-02-06 06:36:00,236 INFO    stdout    :8733 [redirect.py:_pipe_relay():137] relay done done: stdout
2021-02-06 06:36:00,237 INFO    MainThread:8733 [redirect.py:_stop():291] _stop joined: stdout
2021-02-06 06:36:00,237 INFO    MainThread:8733 [redirect.py:_stop():293] _stop rd closed: stdout
2021-02-06 06:36:00,237 INFO    MainThread:8733 [redirect.py:uninstall():228] uninstall done
2021-02-06 06:36:00,237 INFO    MainThread:8733 [redirect.py:uninstall():224] uninstall start
2021-02-06 06:36:00,237 INFO    MainThread:8733 [redirect.py:_stop():279] _stop: stderr
2021-02-06 06:36:00,237 INFO    MainThread:8733 [redirect.py:_stop():285] _stop closed: stderr
2021-02-06 06:36:00,237 INFO    stderr    :8733 [redirect.py:_pipe_relay():121] relay done saw last write: stderr
2021-02-06 06:36:00,237 INFO    stderr    :8733 [redirect.py:_pipe_relay():137] relay done done: stderr
2021-02-06 06:36:00,237 INFO    MainThread:8733 [redirect.py:_stop():291] _stop joined: stderr
2021-02-06 06:36:00,237 INFO    MainThread:8733 [redirect.py:_stop():293] _stop rd closed: stderr
2021-02-06 06:36:00,238 INFO    MainThread:8733 [redirect.py:uninstall():228] uninstall done
2021-02-06 06:36:00,575 INFO    MainThread:8733 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "db7c7ec80e554fcbad1465a5121972f4"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 511
      total_bytes: 511
    }
  }
}

2021-02-06 06:36:02,578 INFO    MainThread:8733 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "187b4b1a003c4fc8801211b3e6ad9440"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 6603
      total_bytes: 6603
    }
  }
}

2021-02-06 06:36:03,951 INFO    MainThread:8733 [wandb_run.py:_show_files():1614] logging synced files
2021-02-06 06:36:03,953 INFO    MainThread:8733 [internal.py:handle_exit():139] Internal process exited
